src/astro/settings.py

```diff
@@ -2,3 +2,13 @@
 
 DEFAULT_SCHEMA = "tmp_astro"
 SCHEMA = conf.get("astro_sdk", "sql_schema", fallback=DEFAULT_SCHEMA)
+
+SNOWFLAKE_STORAGE_INTEGRATION_AMAZON = conf.get(
+    section="astro_sdk", key="snowflake_storage_integration_amazon", fallback=None
```

Comment:
Why do we have a not-None fallback for Google integration but not Amazon?

Reply:
Because otherwise, we would always rely on Amazon storage integration - which is not needed for AWS (we can retrieve `secret_id` and `access_key` from the Airflow connection). Would you like me to add a comment to the code?